src/app/(admin)/bundles/components/Bundle.tsx

```diff
@@ -137,6 +148,16 @@ export const Bundle = ({ bundleId }: { bundleId: number }) => {
 
         <div>
           <Link href={`/bundles/${bundle.id}/edit`}>Edit</Link>
+          &nbsp;
+          |&nbsp;
+          { generateOSEPublicBundleLink(bundle, "en")!==null && (
+            <Link href={{ pathname: generateOSEPublicBundleLink(bundle, "en") }}>Export English</Link>
```

Comment:
Can we get the list of languages and then iterate on them instead of hardcoding this way?

Reply:
done